Install error with 2.8.1

Was just doing an install and got this error:
```
$ npm i @popperjs/core

> @popperjs/core@2.8.1 postinstall .../node_modules/@popperjs/core
> husky install .config/husky

sh: husky: command not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! @popperjs/core@2.8.1 postinstall: `husky install .config/husky`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the @popperjs/core@2.8.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     .../.npm/_logs/2021-02-20T19_12_22_194Z-debug.log
```

I noticed 2.8.1 had just been posted within the hour so I tried installing version 2.7.2 via:
```
npm i @popperjs/core@2.7.2
```
I saw no error with that command.

Cheers,
-W

Same here, and after install husky manualy, the error is `.git can't be found`.

```
$> git:(master) ✗ npm install 

npm ERR! code 127
npm ERR! path /var/www/resources/views/assets/node_modules/@popperjs/core
npm ERR! command failed
npm ERR! command sh -c husky install .config/husky
npm ERR! sh: 1: husky: not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/www/.npm/_logs/2021-02-20T19_27_12_709Z-debug.log

$> git:(master) ✗ npm install husky --save-dev

added 1 package, and audited 1231 packages in 2s

$> git:(master) ✗ npm install   

npm ERR! code 1
npm ERR! path /var/www/resources/views/assets/node_modules/@popperjs/core
npm ERR! command failed
npm ERR! command sh -c husky install .config/husky
npm ERR! /var/www/resources/views/assets/node_modules/husky/lib/commands/install.js:20
npm ERR!         throw new Error(".git can't be found");
npm ERR!         ^
npm ERR! 
npm ERR! Error: .git can't be found
npm ERR!     at Object.install (/var/www/resources/views/assets/node_modules/husky/lib/commands/install.js:20:15)
npm ERR!     at Object.<anonymous> (/var/www/resources/views/assets/node_modules/husky/lib/bin.js:43:19)
npm ERR!     at Module._compile (internal/modules/cjs/loader.js:999:30)
npm ERR!     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
npm ERR!     at Module.load (internal/modules/cjs/loader.js:863:32)
npm ERR!     at Function.Module._load (internal/modules/cjs/loader.js:708:14)
npm ERR!     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
npm ERR!     at internal/main/run_main_module.js:17:47

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/www/.npm/_logs/2021-02-20T19_29_14_868Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -44,8 +44,9 @@
     "build:dev": "NODE_ENV=dev babel src -d tests/visual/dist",
     "build:flow": "flow-copy-source --ignore \"**/*.test.js\" src lib && replace-in-files --string=__DEV__ --replacement=false 'lib/**/*.flow'",
     "build:typescript": "rimraf dist/typescript; flow-to-ts \"src/**/*.js\" --write --inline-utility-types; tsc-silent --project .config/tsconfig.json --createSourceFile .config/createSourceFile.js --suppress @; rimraf \"src/**/*.ts\"",
-    "prepublishOnly": "yarn build",
-    "postinstall": "husky install .config/husky"
+    "prepublishOnly": "yarn build && pinst --disable",
+    "postinstall": "husky install .config/husky",
+    "postpublish": "pinst --enable"
   },
   "prettier": {
     "semi": true,
@@ -101,6 +102,7 @@
     "jest-environment-puppeteer": "^4.4.0",
     "jest-image-snapshot": "^4.3.0",
     "jest-puppeteer": "^4.4.0",
+    "pinst": "^2.1.4",
     "poster": "^0.0.9",
     "prettier": "^2.2.1",
     "pretty-quick": "^3.1.0",

